src/main/kotlin/com/hibob/kotlinBascis/StoreService.kt

```diff
@@ -0,0 +1,78 @@
+package com.hibob.kotlinBascis
+import org.springframework.stereotype.Component
+import java.time.LocalDate
+
+@Component
+class StoreService {
+
+    fun pay(cart: List<Cart>, payment: Payment): Map<String, Check> {
+        val ans = mutableMapOf<String, Check>()
```

Comment:
for this practice that fine, but in general we will avoid using mutableMapOf, we will always prefer to use immutable structures.